File: sephora-backend/Application/Dtos/Delivery/DeliveryDto.cs
namespace CleanArchitecture.Application.Dtos.Delivery;

// TODO: Implement
public class DeliveryDto
{
    public long Id { get; set; }
    public string Address { get; set; } = String.Empty;
    public string Provider { get; set; } = String.Empty;
    
    public string? UserId { get; set; }
    public string? UserName { get; set; }
    public string? UserEmail { get; set; }
    public string? UserPhoneNumber { get; set; }
}

File: sephora-frontend/src/models/delivery/DeliveryDto.ts
export default interface DeliveryDto {
    get id(): number;
    get address(): string;
    get provider(): string;

    get userId(): number;
    get userName(): string;
    get userEmail(): string;
    get userPhone(): string;

    // TODO: Add the following properties to the DeliveryDto interface:
    // get status(): string;
    // get createdAt(): Date;
    // get createdBy(): string;
    // get deliveryDate(): Date;
}
